### PR TITLE
Releasefix 236

### LINKED ISSUE #236: Sub-items aren't displayed properly on GitHub Pages
**Is your feature request related to a problem? Please describe.**
Related to #163 . In the markdown, the sub-items for step 3 are shown properly with squares as bullets: https://github.com/aau-giraf/wiki/blob/master/docs/writing_wiki.md
In GitHub pages they are apparently not treated as sub-items: https://aau-giraf.github.io/wiki/writing_wiki/

**Describe the solution you'd like**
The sub-items under step 3 should be displayed properly on GitHub Pages as they are in the markdown.

**Additional context**
Screenshot from markdown:
image
Screenshot from GitHub Pages:
image

### PR DESCRIPTION
**Pull request by RasmusHRAAU**
Fixes #236

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw610f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw611f20 you have been assigned this. Here's your checklist.

Please use the following checklist:

**Code Documentation Only**

- [ ] It is documented which BLoCs/Screens the current class is related to
- [ ] The responsibility/functionality of the current class is explained. Also regarding the classes it is related to
- [ ] If any unusual decisions are introduced in the code, they reasoned about in the documentation
 
**Wiki Related**

- [ ] There are no dead links or references
- [ ] The markdown renders correctly in MkDocs
- [ ] The content is placed in the correct page/folder/section

You can run MkDocs locally by following [this guide](https://aau-giraf.github.io/wiki/writing_wiki/#running-locally)

File: docs/writing_wiki.md [hunk 1]
```diff
@@ -27,12 +27,14 @@ The wiki can be run locally by doing the following in a terminal:
     * if not, check ``pip3 -V``
 3. Set up a virtual environment (steps assume that your current dir is project root) 
       1. Install virtualenv with ``pip install virtualenv``
-            * else ``pip3 install virtualenv``
+         <ul><li>else ``pip3 install virtualenv``</li></ul>
       2. run ``python -m venv venv``
-            * else ``python3 -m venv venv``
+	      <ul><li>else ``python3 -m venv venv``</li></ul>
```
Comment (antonhpalmer) [inline docs/writing_wiki.md]: wrong indentation

Comment (antonhpalmer) [reply docs/writing_wiki.md]: line 32

**Review by mathiasandresen — CHANGES_REQUESTED** [review]
det burde ikke være nødvendigt med HTML

**Comment by mathiasandresen** [comment]
> 
> 
> det burde ikke være nødvendigt med HTML

Jeg har fået det til at virke med

```markdown
1. Install Python 3.
2. Check that pip is using python3 with ``pip -V``.
    * if not, check ``pip3 -V``
3. Set up a virtual environment (steps assume that your current dir is project root) 
      1. Install virtualenv with ``pip install virtualenv``
        * else ``pip3 install virtualenv``
      2. run ``python -m venv venv``
        * else ``python3 -m venv venv``
      1. source the virtual environment
        * Linux:    ``source venv/bin/activate``
        * Windows:  ``.\venv\Scripts\activate.bat``
4. Install plugins with ``pip install -r requirements.txt`` (if pip is using python3)
    * else ``pip3 install -r requirements.txt``
5. Start local server with ``mkdocs serve``
```

**Comment by mathiasandresen** [comment]
> 
> 
> > det burde ikke være nødvendigt med HTML
> 
> Jeg har fået det til at virke med
> 
> ```gfm
> 1. Install Python 3.
> 2. Check that pip is using python3 with ``pip -V``.
>     * if not, check ``pip3 -V``
> 3. Set up a virtual environment (steps assume that your current dir is project root) 
>       1. Install virtualenv with ``pip install virtualenv``
>         * else ``pip3 install virtualenv``
>       2. run ``python -m venv venv``
> 	    * else ``python3 -m venv venv``
>       1. source the virtual environment
>         * Linux:    ``source venv/bin/activate``
>         * Windows:  ``.\venv\Scripts\activate.bat``
> 4. Install plugins with ``pip install -r requirements.txt`` (if pip is using python3)
>     * else ``pip3 install -r requirements.txt``
> 5. Start local server with ``mkdocs serve``
> ```

hov, formatting er forkert

**Comment by RasmusHRAAU** [comment]
> > det burde ikke være nødvendigt med HTML
> 
> Jeg har fået det til at virke med
> 
> ```gfm
> 1. Install Python 3.
> 2. Check that pip is using python3 with ``pip -V``.
>     * if not, check ``pip3 -V``
> 3. Set up a virtual environment (steps assume that your current dir is project root) 
>       1. Install virtualenv with ``pip install virtualenv``
>         * else ``pip3 install virtualenv``
>       2. run ``python -m venv venv``
>         * else ``python3 -m venv venv``
>       1. source the virtual environment
>         * Linux:    ``source venv/bin/activate``
>         * Windows:  ``.\venv\Scripts\activate.bat``
> 4. Install plugins with ``pip install -r requirements.txt`` (if pip is using python3)
>     * else ``pip3 install -r requirements.txt``
> 5. Start local server with ``mkdocs serve``
> ```

Virker det også når man render med mkdocs, det synes jeg ikke det gjorde da jeg prøvede det

File: docs/writing_wiki.md [hunk 2]
```diff
@@ -27,12 +27,12 @@ The wiki can be run locally by doing the following in a terminal:
...
       1. Install virtualenv with ``pip install virtualenv``
-            * else ``pip3 install virtualenv``
+        * else ``pip3 install virtualenv``
       2. run ``python -m venv venv``
-            * else ``python3 -m venv venv``
-      3. source the virtual environment
-            * Linux:    ``source venv/bin/activate`` 
-            * Windows:  ``.\venv\Scripts\activate.bat``
+        * else ``python3 -m venv venv``
+      1. source the virtual environment
```
Comment (klogeat) [inline docs/writing_wiki.md]: It says 1. source, but should be  3. source. Maybe use a, b and c instead of numbers?

**Review by klogeat — DISMISSED** [review]

**Review by klogeat — APPROVED** [review]

**Review by mathiasandresen — APPROVED** [review]

**Comment by mathiasandresen** [comment]
@RasmusHRAAU husk at slette branch :)

**Pull request merged**